src/main/java/com/bearcat/eventplanner/SecurityConfig.java

```diff
@@ -2,22 +2,27 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.Customizer;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.web.SecurityFilterChain;
 
 @Configuration
 public class SecurityConfig {
 
-    @Bean
+    //@Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         http
-            .authorizeHttpRequests()
-                .requestMatchers("/", "/login").permitAll()
-                .anyRequest().authenticated()
-            .and()
-            .oauth2Login()
-                .loginPage("/login")
-                .defaultSuccessUrl("/home", true);
+            .authorizeHttpRequests(authorizeHttpRequests ->
```

Comment:
I changed it so that the methods in SecurityFilterChain aren't using deprecated versions. At some point the deprecated versions could stop working with future updates to Spring, so I was thinking that leaving them deprecated could possibly [incur technical debt](https://martinfowler.com/bliki/TechnicalDebt.html) in the future.